### PR TITLE
Remove prevent propagation

### PR DESCRIPTION
**Pull request by gareth-amazon**
## Overview
* Remove preventPropagation flag so events are always propagated to all members of a viewport group
* bump major version number due to breaking behavior change

## Tests
[Unit Tests](https://github.com/awslabs/synchro-charts/runs/5346110552?check_suite_focus=true)
[Integ Tests](https://github.com/awslabs/synchro-charts/runs/5346110518?check_suite_focus=true)

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/synchro-charts/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@synchro-charts/core",
   "description": "Synchro Charts",
-  "version": "1.1.1",
+  "version": "2.1.1",
```
Comment (NorbertNader) [inline packages/synchro-charts/package.json:4]: I think we should reset the minor and patch version.

**Review by NorbertNader — APPROVED** [review]

File: lerna.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
         "packages/*"
     ],
     "npmClient": "yarn",
-    "version": "1.1.1",
+    "version": "2.0.0",
```
Comment (NorbertNader) [inline lerna.json:6]: Looks like this one is also bumping to 2.0.0 https://github.com/awslabs/synchro-charts/pull/123/files
We might want to send these out in the same release, so skip the version bump in both of these and send out a separate PR with the version bump containing both.

**Review by NorbertNader — APPROVED** [review]

File: lerna.json (hunk 2 above)
Comment (NorbertNader) [inline lerna.json:6]: Let's revert the version bump here and release 2.0.0 with this PR https://github.com/awslabs/synchro-charts/pull/123

**Review by diehbria — APPROVED** [review]
